Add control example (#110)
eclipsesource/jsonforms2-website: src/pages/examples/control.mdx

---
name: Control
route: /examples/control
---

import { 
  Control, 
  ExtendedControl 
} from '../../components/examples/Control'

# Control

> On this page you can see the "normal" control types JSON Form provides.

Reviewer: ```suggestion
These are the default controls JSON Form provides:
```

<Control />

> Some controls can be configured via UI schema options

Reviewer: ```suggestion
Controls can be configured via `options`, see the following examples:
```
